### PR TITLE
:heart:  Likes involvement - Like Counter

### PR DESCRIPTION
**Pull request by SimonGideon**
## :building_construction: In this PR I:

- [x] The Involvement API to show the item likes.
- [x] Remember that your page should make only 2 requests:
- [x] One to the base API.
- [x] And one to the Involvement API.

Check the respective files :clinking_glasses:

### CODE REVIEW

**Review by Zia123456 — COMMENTED** [review]
# Suggestions
- follow HTML and CSS best practice
# Highlights
- professional readme file
- Nice design
cheers 🍷

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # .gitignore
-**/node_modules
\ No newline at end of file
+**/node_modules
+.env
```
[the remark this answers is not in the document]
Comment (SimonGideon) [reply .gitignore:3]: :muscle:  Thanks for the review @Zia123456 for the review. :slightly_smiling_face:

File: src/index.html [hunk 2]
```diff
@@ -4,24 +4,32 @@
...
     <title>Royal Meals</title>
     <body>
     <header>
       <nav>
-        <img src="../logo.png" alt="" srcset="">
-        <li>Available foods<span id="countfood"></span></li>
-        <li>Menu</li>
-        <li>More</li>
-      </ul>
```
[the remark this answers is not in the document]
Comment (SimonGideon) [reply src/index.html:16]: :slightly_smiling_face: @Zia123456 I saw it better to customize the text elements from the wireframe this freedom is given from the  project instructions for it to match my chosen API topic

File: src/index.html [hunk 3]
```diff
@@ -4,24 +4,32 @@
...
+          <li>Available foods<span id="countfood"></span></li>
+          <li href="#" >Menu</li>
+          <li>More</li>
+        </ul>
+        <div class="searchBox">
+          <input class="searchInput"type="text" name="" placeholder="Search">
+          <button class="searchButton" href="#">
+            <i class="fa-solid fa-magnifying-glass"></i>
+          </button>
+      </div>
```
[the remark this answers is not in the document]
Comment (SimonGideon) [reply src/index.html:24]: :slightly_smiling_face: :muscle:  Thanks for the second eye on this. I have implemented this as suggested.

File: src/modules/fetchLikes.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-undef */
+/* eslint-disable camelcase */
+/* eslint-disable no-unused-vars */
+const ProductList = document.querySelector('#meals-content');
+const likedMeal = async (id) => {
+  try {
+    const resp = await fetch(
+      'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/o8FdGibCYqeybnMSFY1p/likes/',
```
[the remark this answers is not in the document]
Comment (SimonGideon) [reply src/modules/fetchLikes.js:8]: :slightly_smiling_face:  It was hard for me to implement the `dotevn` in jest to hide the API and access it with the process.

File: src/styles/styles.css [hunk 5]
```diff
@@ -1,7 +1,10 @@
-@import url('https://fonts.googleapis.com/css2?family=Play:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Dancing+Script:wght@400;700&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100&display=swap');
 * {
     margin: 0;
 }
+html {
```
[the remark this answers is not in the document]
Comment (SimonGideon) [reply src/styles/styles.css:5]: :+1: Noted

**Pull request merged**